frontend/src/utils/themes.ts

```diff
@@ -50,6 +50,9 @@ const paletteBase = {
         light: lighten("#d0ffb2", 0.1),
         contrastText: "#000",
     },
+    iconButtonWhite: {
```

Comment:
Maybe rename this? This does not indicate what type of button it's and the colour in the name might be unnecessary?